packages/spark-core/components/stepper.js

```diff
@@ -8,7 +8,7 @@ import {
 } from './tabs';
 import { carousel } from './carousel';
 
-const resetSlider = (steps, activeClass, slider) => {
+const resetSlider = (steps, slider) => {
```

Comment:
Its not needed anymore cause i'm applying a different class to the slider when its active. I could rename it to something like activeSliderClass, but since the function is already tied to the slider directly, i didnt see much value in abstracting.

Reply:
Okay, I agree there's not a strong need to be abstracted.